components/HeroListFeature/HeroListFeature.js

```diff
@@ -2,57 +2,67 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 import { CustomLink } from '..';
-import { DefaultCard, HeroCardGrid } from 'ui-kit';
+import { Box, CardGrid, DefaultCard, RowCard } from 'ui-kit';
 import { getURLFromType } from 'utils';
+import dropRight from 'lodash/dropRight';
```

Comment:
Nitpick: Can we move this to the top with the `PropTypes` import? We've been keeping the third-party vs. local packages grouped separately.